sycl/test-e2e/ESIMD/performance_tests/BitonicSortK.cpp

```diff
@@ -0,0 +1,15 @@
+//==---------------- BitonicSortK.cpp  - DPC++ ESIMD on-device test --------==//
```

Comment:
Can we rename the folder to `PerformanceTests` to match the [general SYCL folder name](https://github.com/intel/llvm/tree/sycl/sycl/test-e2e/PerformanceTests)?